examples/dump_media/main.py

```diff
@@ -0,0 +1,102 @@
+from pathlib import Path
```

Comment:
please run a `black  examples/dump_media/main.py` it will wrap the long lines.

Reply:
done